### ISSUE TITLE
Design sprints: overview

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/master-reference#72]
Design sprints:

- happen over the last two weeks of the 16-week programme;
- are intended to be a one-week engagement with real clients; 
- will usually lead to some sort of post-graduate project (for at least two people);
- tend to consist of two (sometimes three)-person teams;
- usually start with a one-day workshop with clients;
- workshops will sometimes happen on a Saturday, in which case the preceding Friday or following Monday are usually considered a substitute rest day;
- usually end with a pitch event from which post-graduate projects will be selected;
- where resources allow, teams can expect support with project management, UI/UX, user testing and code reviewing;
- where we have more than two engagements, we will split the group into two and run two design sprints in parallel up to a total of four engagements over two weeks. 
- when we have more than four engagements, we will need to propose a third week of design sprints.

In summary, we currently need two weeks for design sprints, but may want to ask for a third week in future.

**Comment by shiryz** [comment]
Going to close this issue, because client projects have evolved since this was opened.
if still relevant and not recorded anywhere else, please reopen
